### PR TITLE
FINERACT-689

### PR DESCRIPTION
**Pull request by palanisami**
## Description
FINERACT-689

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Commit message starts with the issue number from
 FINERACT-689

- [ ] Coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions have been followed.

- [ ] API documentation at https://github.com/apache/fineract/blob/develop/api-docs/apiLive.htm has been updated with details of any API changes.

- [ ] Integration tests have been created/updated for verifying the changes made.

- [ ] All Integrations tests are passing with the new commits.

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the list.)

Our guidelines for code reviews is at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/interestratechart/data/InterestRateChartSlabDataValidator.java [hunk 1]
```diff
@@ -249,7 +249,8 @@ public void validateChartSlabsUpdate(final JsonElement element, final DataValida
         if (this.fromApiJsonHelper.parameterExists(annualInterestRateParamName, element)) {
             final BigDecimal annualInterestRate = this.fromApiJsonHelper.extractBigDecimalNamed(annualInterestRateParamName, element,
                     locale);
-            baseDataValidator.reset().parameter(annualInterestRateParamName).value(annualInterestRate).notNull().zeroOrPositiveAmount();
+            //Change made to not to allow interest rate more than 100%
+            baseDataValidator.reset().parameter(annualInterestRateParamName).value(annualInterestRate).notNull().zeroOrPositiveAmount()value(annualInterestRate).notNull().zeroOrPositiveAmount().notGreaterThanMax(new BigDecimal(100));
```
Comment (angelboxes) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/interestratechart/data/InterestRateChartSlabDataValidator.java:253]: I think there's something missing or extra around zeroOrPositiveAmount()value, there are some repeated function calls here, it should be fixed just by removing them.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/interestratechart/exception/InterestRatePeriodValidateException.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import org.apache.fineract.infrastructure.core.exception.AbstractPlatformResourceNotFoundException;
```
Comment (angelboxes) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/interestratechart/exception/InterestRatePeriodValidateException.java:2]: license header is missing in this file

**Review by angelboxes — CHANGES_REQUESTED** [review]
I made some comments since apparently the build was failing.

**Comment by vorburger** [comment]
@palanisami will you address the feedback from @angelboxes and make this PR pass the build?

**Comment by vorburger** [comment]
@palanisami would you like to address the feedback?  We would otherwise like to close this PR in another 2 weeks (after 1 month), just so we don't have too many stale open ones; thanks for understanding.

**Comment by vorburger** [comment]
Let's close this now, just to have less open PRs; you can open a new PR with this when you resolved the feedback and conflicts.

**Pull request closed without merging**